### ISSUE TITLE
Ambience clipping as you orbit around areas sounds horrible

### ISSUE THREAD

**Issue by Mothblocks** [tgstation/tgstation#85568]
Round 233280

https://github.com/user-attachments/assets/fab66e9c-0911-41f7-a608-34abb5f5e268

**Comment by Potato-Masher** [comment]
Honestly this has been around for quite some time with all the ambiance changes.

**Comment by Sadboysuss** [comment]
> with all the ambiance changes. 

The recent addition of 5 maint ambiences has nothing to do with it.
It's been in the game forever

**Comment by Potato-Masher** [comment]
Perhaps it has nothing to do with it, but it is far more obvious than it once was.

**Comment by MrMelbert** [comment]
#84479 related. IIRC ghosts used to never change ambience from moving areas so it's only noticeable now because it was broken before

**Comment by Mothblocks** [comment]
Yeah I figure this has been around for a while I haven't logged on in a bit
